src/scancode/cli.py

```diff
@@ -724,7 +724,7 @@ def scancode(ctx, input,  # NOQA
         # TODO: this is weird: may be the timings should NOt be stored on the
         # codebase, since they exist in abstract of it??
         codebase.timings.update(setup_timings)
-
+        codebase.timings['timestamp'] = scan_start
```

Comment:
IMHO the scan_start should be its own attribute on the Codebase. This is not a "timing" proper.